public/index.html

```diff
@@ -50,7 +50,8 @@
         m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
         })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
         ga('create', 'UA-133820299-2', 'auto');
-
+        ga('set', 'page', location.href);
+        ga('send', 'pageview');
```

Comment:
I could easily be wrong, but this might cause double page views.  Maybe keep an eye on traffic and if it spikes hard after this release I would circle back to this change.

Reply:
I'm not sure that it will double call but probably it's not needed.

Reply:
https://github.com/Coding-Coach/find-a-mentor/pull/570/commits/3c89d4d803fee5507fffd94c44caab9a34f02498